src/parser/mod.rs

```diff
@@ -7708,6 +7708,9 @@ impl<'a> Parser<'a> {
 
         while let Some(option) = self.parse_plain_option()? {
             options.push(option);
+            if self.dialect.supports_comma_separated_create_table_options() {
+                let _ = self.consume_token(&Token::Comma);
+            }
```

Comment:
Thinking since the syntax wouldn't conflict with other dialects we can skip the dialect option and have the parser always consume the comma if it shows up?